components/SelectedArticleScreen.jsx

```diff
@@ -1,15 +1,20 @@
 import { useParams } from 'react-router-dom';
-import styles from "./HomeScreen.module.css";
+import { useState } from "react";
+import styles from "./SelectedArticleScreen.module.css";
+import Comments from "./sub-components/Comments"
 import SelectedArticle from './sub-components/SelectedArticle';
 import NavBar2 from "./sub-components/NavBar2";
 import Header from "./sub-components/Header";
 
-const SelectedArticleScreen = ({votes, loading, setLoading}) => {
+const SelectedArticleScreen = ({votes, isLoading, setIsLoading}) => {
```

Comment:
Same question as above, where are these props coming from and what purpose do they have as I can see no good reason for them to be here.